tests/pwm/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+#![no_std]
+
+use embedded_hal::Pwm;
+use riot_wrappers::pwm::{HertzU32, PWMChannel, PWMDevice, PWMMode};
```

Comment:
```suggestion
use riot_wrappers::pwm::{HertzU32, PWMDevice, PWMMode};
```

The type gets used but not named -- there's a warning about this.